getting-started/setting-env.md

```diff
@@ -0,0 +1,120 @@
+
+```sh
+wasmcli config chain-id testing
+wasmcli config trust-node true
+# if connecting to local node, wasmcli config node http://localhost:26657
+wasmcli config node https://rpc.demo-10.cosmwasm.com:443
+wasmcli config output json
+wasmcli config indent true
+# this is important, so the cli returns after the tx is in a block,
+# and subsequent queries return the proper results
+wasmcli config broadcast-mode block
+
+# check you can connect
+wasmcli query supply total
+wasmcli query staking validators
+wasmcli query wasm list-code
+
+# add more wallets for testing
+wasmcli keys add fred
+>
+{
+  "name": "fred",
+  "type": "local",
+  "address": "cosmos1avdvl5aje3zt0uay40uj6l9xtqtlqhduu84nql",
+  "pubkey": "cosmospub1addwnpepqvcjveqepq34x59fnmygdy58ag7zwu8gefgsprq9th38nxzptpgszc3rkve",
+  "mnemonic": "hobby bunker rotate piano satoshi planet network verify else market spring toward pledge turkey tip slim word jaguar congress thumb flag project chalk inspire"
+}
+wasmcli keys add bob
+wasmcli keys add thief
+```
+
+You need some tokens in your address to interact. If you are using local node you can skip this step.
+Requesting tokens from faucet:
+
+```sh
+JSON=$(jq -n --arg addr $(wasmcli keys show -a fred) '{"ticker":"COSM","address":$addr}') && curl -X POST --header "Content-Type: application/json" --data "$JSON" https://faucet.demo-10.cosmwasm.com/credit
+JSON=$(jq -n --arg addr $(wasmcli keys show -a thief) '{"ticker":"COSM","address":$addr}') && curl -X POST --header "Content-Type: application/json" --data "$JSON" https://faucet.demo-10.cosmwasm.com/credit
+```
+
+## Setup Node REPL
+
+Beyond the standard CLI tooling, we have also produced a flexible TypeScript library [CosmJS](https://github.com/CosmWasm/cosmjs), which runs in Node.js as well as in modern browsers and handles queries and submitting transactions.
+Along with this library, we produced [@cosmjs/cli](https://www.npmjs.com/package/@cosmjs/cli), which is a super-charged
+Node console. It supports `await`, does type checking for helpful error messages, and preloads many CosmJS utilities.
+If you are comfortable with the Node console, you will probably find this easier and more powerful than the CLI tooling.
+
+Full usage and installation [instructions are on the README](https://github.com/CosmWasm/cosmjs/tree/master/packages/cli), but here is a short version for those who want to run from source:
+
+```sh
+npx @cosmjs/cli --init https://raw.githubusercontent.com/CosmWasm/cosmjs/v0.22.0/packages/cli/examples/helpers.ts
+```
+
+Using the REPL:
+
+```js
+// Create or load account
+const mnemonic = loadOrCreateMnemonic('fred.key')
+mnemonicToAddress('cosmos', mnemonic)
+
+const { address, client } = await connect(mnemonic, {})
+address
+
+client.getAccount()
+// if empty - this only works with CosmWasm
+hitFaucet(defaultFaucetUrl, address, 'COSM')
+client.getAccount()
+```
+
+## Run Local Node (optional)
```

Comment:
Node REPL instructions only work for `demo-10`. We need to add instructions for connecting node repl to local net.

Reply:
You can leave that for later, or maybe @webmaster128 can comment.

Reply:
I suggest copying this file to the docs directory (different sub-directory per network):
https://github.com/CosmWasm/cosmjs/blob/master/packages/cli/examples/helpers.ts

It should not be necessary to update CosmJS when networks change

Reply:
I think this file fits better to testnet repo. I will take care of this later

Reply:
Jap, I agree. There we have the correct folders already